CHANGES.rst

```diff
@@ -153,6 +153,8 @@ Cubeviz
 
 - Broadcast snackbar message to user when sonification of a data cube completes. [#3647]
 
+- Update sonified layer icon to distinguish from other layers in legend, plot options, and data menu. [#3660]
+
```

Comment:
I don't think 4.2 has the concept of sonified layers (introduced in #3430 milestoned to 4.3), so can this be remilestoned there and the PR number added to the existing changelog entry?